.github/workflows/gh-pages.yaml

```diff
@@ -0,0 +1,122 @@
+name: Github Pages
+on:
+  push:
+    branches: [ 'master' ]
+jobs:
+  publish:
+    name: Publish stats, tables and guides
+    runs-on: ubuntu-20.04
+    env:
+      PAGES_DIR: __pages
+    steps:
+      - name: Install Deps
+        uses: mstksg/get-package@master
+        with:
+          apt-get: cmake ninja-build libopenscap8 libxml2-utils expat xsltproc python3-jinja2 python3-yaml python3-setuptools ansible-lint python3-github bats python3-pytest python3-pytest-cov python3-pip
+      - name: Install deps python
+        run: pip3 install json2html
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Build
+        run: cmake .. -G Ninja
+        working-directory: ./build
+      - name: Build Guides and Mapping Tables
+        run: ninja -j2
+        working-directory: ./build
+      - name: Build Statistics
+        run: ninja html-stats html-profile-stats -j2
+        working-directory: ./build
+      - name: Generate HTML pages
+        run: |-
+          mkdir $PAGES_DIR
```

Comment:
No way! Please create a script in the project!

Reply:
Fixed by rebase. I didn't spend much time on creating the script as it was a straight copy from the jenkins job. I particularly don't like to host this script in the project as it's not very useful for others and I'm afraid it would take too much time to turn into something reusable.

Reply:
On the contrary, it is quite useful! First of all it can be tested and debugged without GH Actions involvement. And then, as the consequence, it could be used by some other CI solution if desired (like GitLab). By keeping the actions code to the minimum we make the project less dependent on GH.

Also if, even in this form, it is good enough to provide some value for project developers why we should be hesitant about adding it to the project?